CHANGELOG.rst

```diff
@@ -44,6 +44,9 @@ Removals
   Use `~sunpy.map.GenericMap.shift_reference_coord` instead. (`#6437 <https://github.com/sunpy/sunpy/pull/6437>`__)
 - ``sunpy.util.scraper`` has been removed. Use `sunpy.net.scraper` instead. (`#6438 <https://github.com/sunpy/sunpy/pull/6438>`__)
 - ``sunpy.image.coalignment`` has been removed. Use `sunkit_image.coalignment` instead, which contains all the same functionality. (`#6440 <https://github.com/sunpy/sunpy/pull/6440>`__)
+- ``sunpy.io.cdf`` sub-module has been removed,as it was designed for internal use.
+- ``sunpy.io.file_tools`` sub-module has been removed,as it was designed for internal use.
+- ``sunpy.io.jp2`` sub-module has been removed,as it was designed for internal use.
```

Comment:
This should be a new changelog entry and not one added to the pre-existing changelog.
See https://github.com/sunpy/sunpy/blob/main/changelog/README.rst for more information.